packages/storybook/elements-stencil-docs.json

```diff
@@ -4278,6 +4278,22 @@
           "optional": false,
           "required": false
         },
+        {
+          "name": "for",
+          "type": "string",
+          "mutable": false,
+          "attr": "for",
+          "reflectToAttr": false,
+          "docs": "Id of the associated form control",
+          "docsTags": [],
+          "values": [
+            {
+              "type": "string"
+            }
+          ],
+          "optional": false,
+          "required": false
```

Comment:
Just curious about this config: Doesn't `"optional": false` imply that it is required? Or how is optional and required to be understood in this context?

Reply:
That's actually not quite clear to me either. The generation of this file is done by stencil so maybe the documentation has some infos: https://stenciljs.com/docs/doc-generation